README.md

```diff
@@ -81,6 +81,26 @@ The Management DB has been tested with MySQL and H2, you should be able to use a
 by hibernate, it is highly recommended that the management DB resides on the same physical machine as the application to 
 eliminate any possibility of being unreachable and lower latency hence better performance.
 
+# MySQL 8 Support
+
+## Public Key Retrieval
+the property `allowPublicKeyRetrieval` should be set to true to avoid the issue `Unable to connect: Public Key Retrieval is not allowed`.
+this property should be added to these properties:
+- `debezium.extraParameters` ( extra parameters given to Debezium)
```

Comment:
@icrc-fdeniger, this means `&allowPublicKeyRetrieval=true` should be what's appended to the `debezium.extraParameters` parameter?

Reply:
@icrc-fdeniger, sorry I left my comment in draft and forgot to hit the `Comment` button. Could we add some test cases to cover this enhancement?

Reply:
Sure. Is there any existing test on Debezium configuration so that I complete it ?

Reply:
There is none at the moment which is not ideal. Providing one just for this change will make the changes here runnable and demonstrate-able in a test environment and this will also setup a platform/test class in which to test other configs. @icrc-fdeniger, can do tests with a similar setup as of the [OauthProcessorTest.java](https://github.com/openmrs/openmrs-eip/blob/master/commons/src/test/java/org/openmrs/eip/camel/OauthProcessorTest.java)

Reply:
Sure but I won't have any time for that currently.
I tried to figure out how to add this simple test but no so easy currenlty ( Docker is required)
Is it possible to add a task for that and accept this PR as is ?